Source/SIMPLib/Common/TemplateHelpers.cpp

```diff
@@ -150,16 +152,16 @@ IDataArrayWkPtr CreateNonPrereqArrayFromTypeEnum::operator()(AbstractFilter* f,
 }
 
 // -----------------------------------------------------------------------------
-IDataArrayShPtr CreateArrayFromArrayType::operator()(AbstractFilter* f, const size_t& numTuples, const QVector<size_t>& compDims, const QString& arrayName, bool allocate,
+IDataArrayShPtr CreateArrayFromArrayType::operator()(AbstractFilter* f, const size_t& numTuples, const std::vector<size_t>& compDims, const QString& arrayName, bool allocate,
                                                      const IDataArrayShPtr& sourceArrayType)
 {
   CreateArrayFromArrayType classInstance;
-  QVector<size_t> tupleDims(1, numTuples);
+  std::vector<size_t> tupleDims = {numTuples};
```

Comment:
I don't believe the `=` is necessary.  Upon checking the constructors on cppreference, they choose not to include `=`.